core/gather/gatherers/inputs.js

```diff
@@ -63,6 +63,18 @@ function collectElements() {
       return [...labelElToArtifact.keys()].indexOf(labelEl);
     });
 
+    let preventsPaste;
+    const isTextInput =
+      inputEl.type === 'password' ||
+      inputEl.type === 'text' ||
+      inputEl.type === 'search' ||
+      inputEl.type === 'tel' ||
+      inputEl.tagName === 'TEXTAREA' ||
+      (inputEl.inputMode && inputEl.inputMode !== 'none');
```

Comment:
Is this an exhaustive list? Where did you get it from?

Reply:
It is not. Missing anything?

Reply:
Also could make sense to just check every input or textarea element. Shouldn't really ever be a purpose to prevent paste even for eg. a date input...

Reply:
> Also could make sense to just check every input or textarea element. Shouldn't really ever be a purpose to prevent paste even for eg. a date input...

This actually makes more sense. The audit description isn't specific to text elements.